pic32/cores/pic32/WInterrupts.c

```diff
@@ -263,12 +265,22 @@ void __attribute__((nomips16,at_vector(_EXTERNAL_0_VECTOR),interrupt(IPL4SRS)))
 void __attribute__((interrupt(),nomips16)) ExtInt0Handler(void)
 #endif
 {
-
+	VEC_ENT_CODE(EXT_INT0,INT0IF,PIN_INT0);
```

Comment:
Note that the situation is only for NON edge triggered IRQs,
It is how all other hardware that is LEVEL based works....
I have situations where an IRQ is cleared while another comes in, and the line will remain in the state that triggered the ISR in the first place. By the time this part of the code is reached, the line should be readable. The case for a race should be automatically handled as well, since we clear on entry.
It will not affect edge ones at all.